### ISSUE TITLE
HCM: Apply missing SelectedItem background color and SelectedItemText text color for Checkbox and Radio Button in selected state

### ISSUE THREAD

**Issue by alionazherdetska** [swisspost/design-system#4293]
**Description:**
The SelectedItem background color and SelectedItemText text color are not being applied to checkbox and radio button in their selected state in high contrast mode.

![Image](https://github.com/user-attachments/assets/422cd925-78a4-45d8-8d2a-e6f0c940624c)
